src/src_main/crypto_tc.c

```diff
@@ -391,8 +427,11 @@ int32_t Crypto_TC_ApplySecurity(const uint8_t* p_in_frame, const uint16_t in_fra
             return CRYPTO_LIB_ERR_NULL_CIPHERS;
         }
 
-        if(sa_ptr->est == 0 && sa_ptr->ast == 1 && sa_ptr->acs !=NULL && *(sa_ptr->acs) == CRYPTO_MAC_CMAC_AES256 \
-            && *(sa_ptr->acs) == CRYPTO_MAC_HMAC_SHA256 && *(sa_ptr->acs) == CRYPTO_MAC_HMAC_SHA512 &&
+        if(sa_ptr->acs == NULL) printf("OOPS NULL POINTER!\n");
+        else printf("ACS: %02x\n", *sa_ptr->acs);
+
```

Comment:
Cleanup debug statements (that are not surrounded by #ifdef DEBUG)

Reply:
If this is an actual error case, add error code return for it.

Reply:
These are encased in debug #ifdefs now.
It is not an actual error case.  It was just being used for debugging.